### ISSUE TITLE
[Demo] Sending money not working

### ISSUE THREAD

**Issue by steph643** [sakurity/securelogin#36]
I've installed the native app and here is what I get when using the demo to send $123 to "aaa":
![image](https://user-images.githubusercontent.com/7039479/28327479-d7e1fe6e-6be3-11e7-9dc0-28065b2bdfe1.png)

**Comment by andrewda** [comment]
I'll try to take a look at this. Are you using the Ruby or Node.js demo?

**Comment by steph643** [comment]
Node.js

**Comment by andrewda** [comment]
Alright that's mine, so I'll take a peak at why it's gone funky.

**Comment by andrewda** [comment]
Yup looks like there were some protocol changes that I was unaware of – fixing now :)

**Comment by andrewda** [comment]
Alrighty should be good to go now! Thanks for reporting this btw, wouldn't have caught it :)

http://passport-securelogin.herokuapp.com/
